projects/swift-protobuf/fuzz.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
```

Comment:
Is it possible to configure things so we can have a tester for each format (binary, json, textformat)?

Reply:
It is certainly possible.
I think the best way would be to get the fuzz targets to be merged in the upstream repository
And then have a way to build them specifically (so that they are not in regular builds), knowing where the output fuzzers will be and what their names will be so that we can copy them in some other directory...

Reply:
Towards that goal, I'd like to rename this source file `fuzz_binary.swift` so that we have a pattern for adding JSON and TextFormat fuzzers in the future.

Reply:
Done

Reply:
> I think the best way would be to get the fuzz targets to be merged in the upstream repository
> And then have a way to build them specifically (so that they are not in regular builds), knowing where the output fuzzers will be and what their names will be so that we can copy them in some other directory...

So get targets into apple/swift_protobuf for the binaries?

Reply:
Yes cf https://google.github.io/oss-fuzz/advanced-topics/ideal-integration/#fuzz-target

@inferno-chromium what do you think about this integration of a first swift project ?